docs/release-schedule.md

```diff
@@ -0,0 +1,9 @@
+Maintenance period
+==================
+
+| Version | Release Date | Maintenance End | End of Life |
+| ------- | ------------ | --------------- | ----------- |
+| v3.x    | 2021-10-14   | 2022-10-01      | 2023-04-01  |
+| v4.x    | 2022-04-30   | 2022-04-01      | 2023-10-01  |
+| v5.x    | 2022-10-31   | 2022-10-01      | 2024-04-01  |
```

Comment:
hm, versions 4 and 5 are scheduled to be maintained for... negative thrity days? ;-)